pkg/api/types.go

```diff
@@ -1301,6 +1301,16 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// NodeDaemonPorts lists ports opened by daemons running on the Node
+type NodeDaemonPorts struct {
+	// KubeletPort on which Kubelet is listening
+	KubeletPort int `json:"kubeletPort,omitempty"`
+	// KubeletReadOnlyPorty only port open on Kubelet
+	KubeletReadOnlyPort int `json:"kubeletReadOnlyPort,omitempty"`
```

Comment:
Based on the latest updates in https://github.com/kubernetes/kubernetes/issues/12968 I think we should remove the KubeletReadOnlyPort var here.